client/my-sites/importer/section-import.jsx

```diff
@@ -95,7 +95,17 @@ class SectionImport extends Component {
 	handleStateChanges = () => {
 		this.props.siteImports.map( ( importItem ) => {
 			const { importerState, type: importerId } = importItem;
-			this.trackImporterStateChange( importerState, importerId );
+			let eventProps = {};
+
+			// Log more info about upload failures
+			if ( importerState === appStates.UPLOAD_FAILURE ) {
+				eventProps = {
+					error_code: importItem.errorData.code,
+					error_type: importItem.errorData.type,
```

Comment:
We could also add file size here to see what people try upload.

Reply:
This would be nice indeed! We could decide how we adjust the upload limit based on this information. If we see a lot of very large uploads failing, we could also consider implementing something like chunked and resumable uploads.

Reply:
Good addition for another PR!